README.md

```diff
@@ -1,13 +1,32 @@
 # Uppy.io
 
-Website and documentation for [uppy](https://uppy.io/).
+Website and documentation for [uppy.io](https://uppy.io).
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern
-static website generator.
+Code for [uppy.io/blog](https://uppy.io/blog) lives in this repo.  
+Code for [uppy.io/docs](https://uppy.io/docs/quick-start) lives in
+[github.com/transloadit/uppy](https://github.com/transloadit/uppy).
 
 ## Developer guide
 
-### Installation
+If you want to edit [uppy.io/blog](https://uppy.io/blog), just edit the files in
+this repo.
+
+If you want to edit [uppy.io/docs](https://uppy.io/docs/quick-start), then you
+should:
+
+1. Clone both the **uppy** repo and **uppy.io** repo.
+
+2. Then, from your **/uppy.io** repo, run:
+
+   ```sh
+   ln -s /path-to-your-uppy-repo/docs docs
+   ```
+
+   Now, treat **/uppy/docs** as your working directory, but run the build/lint
+   scripts from your **/uppy.io** directory - **/uppy.io** will pick up your
+   changes.
```

Comment:
I don't really understand the "treat as your working directory"
```suggestion
   Now, when running the build/lint scripts from the root of your `transloadit/uppy.io` clone, it will pick up your local changes on the `docs/` folder. Note that changes in the `docs/` changes should be submitted to the `transloadit/uppy` repository.
```

Reply:
"treat /uppy/docs as your working directory" means "/uppy/docs is a folder where you will be editing files, then committing the files, and then pushing the files".